### ISSUE TITLE
Recalculate TOD when FL changes

### ISSUE THREAD

**Issue by StephN-B** [flybywiresim/aircraft#8671]
### Aircraft Version

Stable

### Description

I have noticed that the TOD position is not recalculated if there is a change to the assigned FL during flight.
## SCENARIO
- Plan and perform a flight, e.g.: 
EDDM/08R GIVM6E GIVMI Y101 INBED DCT REKDI DCT LOHRE DCT ADKUV DCT DENUT L608 LOGAN LOGA2H EGLL/27L
-  Climb to assigned FL, e.g.: FL340 -> TOD is calculated correctly and also displayed on ND
-  During flight, set new FL, e.g. FL380 

## CURRENT BEHAVIOUR
TOD position remains as is

## EXPECTED BEHAVIOUR
TOD needs to be recalculated and moved closer to current position

### Discord Username (optional)

StephN-B

**Comment by BlueberryKing** [comment]
Hi, the T/D should already be recalculated when the cruise level changes, I'll try to reproduce this issue.

**Comment by StephN-B** [comment]
I just tried again, this time on a different route, and I have to agree with you that this time around it does work for me.
It may be an issue limited to a specific scenario, for example changing the FL close to the current T/D, or less than x waypoints between current position and the T/D.

I suggest that this ticket can be closed, as the functionality already has been implemented, and if I find a way to recreate the issue I will created a bug ticket instead.
